kube_apiserver_metrics/assets/dashboards/overview.json

```diff
@@ -0,0 +1,809 @@
+{
+  "title": "Kubernetes API Server - Overview",
+  "description": "",
```

Comment:
Please add a description similar to other dashboards. For example:

`"## Airflow\n\nThis dashboard provides a high-level overview of your Airflow instances so you can monitor metrics related to DAGs, tasks, pools and executors.\n\nFor further reading on monitoring Airflow, see:\n\n- [Official Airflow integration docs](https://docs.datadoghq.com/integrations/airflow/#data-collected)\n\nClone this template to make changes and add your own graphs and widgets."`

Reply:
Thanks. I forgot about it.
I added a description in a new commit https://github.com/DataDog/integrations-core/pull/9757/commits/3f8d0a03ea6d6fdbd8afd056f9a94e4524e2bd42